man/adoc/bpftrace.adoc

```diff
@@ -188,6 +188,17 @@ Print bpftrace version information.
 Enable verbose messages.
 For more details see the <<Verbose Output>> section.
 
+=== Program Options
+
+You can also pass custom options to a bpftrace program/script itself via positional or named parameters.
+All positional and named parameters should come after a double dash e.g.
+```
+# bpftrace -e 'BEGIN { ... }' -- p1 --aa=20 --bb p2
+```
+In this example there are two positional parameters (p1, p2) and two named parameters (aa - set to 20 and bb - set to true).
```

Comment:
"aa - set to 20 and bb - set to true" has very confusing punctuation. I would just use a full sentence, e.g. "`aa`, which is set to 20, and `bb`, which is set to true".

(Consider using backticks for the variables to make them stand out there also)